lab_51.md

```diff
@@ -84,15 +87,15 @@ What was the result of the group following the instructions?  Were there any dif
 
   ![Dots](images/dots.png)
 
-## Part 2 -  Reflection
+## Part 3 -  Reflection
 
 1. What was the result of following the instructions?  
 2. Does this seem like the most efficient way to achieve this goal?  
 3. What would happen if the shape needed to be changed?  
 4. What problems could that cause?
 5. What might be a better or more efficient way to accomplish the goal of these instructions?
 
-## Part 3 - The same but different
+## Part 4 - The same but different
 
 Assign each member of your group a different number from 1 to 6.  Each of the six students in your group should then follow these instructions.  Each student should then follow his or her instructions, completing each corresponding step together.  (For example, everybody should complete their step ii. at the same time.) Everyone should work on the same grid.
```

Comment:
Not one of the changes you made, but the instructions in this paragraph are also not inclusive. (Change "his or her" to "their.") Also, you changed the Roman numerals to a section number elsewhere. Should that have been done here?